airbyte-integrations/connectors/source-shopify/source_shopify/schemas/product_variants.json

```diff
@@ -121,8 +121,17 @@
             }
```

Comment:
Based on the sample you provided, the control version has field `image_id` while the target has `image`. Is this expected? Should we document this as well?

Reply:
No, added the handling of that, we basically need to have `image_id` at the root level, but `graphql` nests it in `"image":{"image_id": "asdfasdf"}`. We don't have this populated for our sandbox, therefore it was `Null` for the data and wasn't processed by far. Thanks for the notice! 

Corrected. Updated the dedicated `unit_test` to reflect this and updated the branch.